### PR TITLE
Read me

### PR DESCRIPTION
**Pull request by AlexSkowronski2**
Made some changes obviously:
- added further developments section
- Usage section, with how to use and example screenshots

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,13 +22,47 @@ These take the form of:
...
 
-## Technologies
+## Project Roadmap
+
+![Screenshot_Excalidraw](https://user-images.githubusercontent.com/115073814/217223449-26e6a315-d01c-4a11-a743-ceee6c4e60d7.png)
+
+### Summary of above roadmap:
+
+1. Kafka Consumer in EC2 takes logs from Kafka Data stream every second
+2. Parses through logs and cleans data, passes into SQLite table ready for querying for Live Dashboard, displays information on current ride and recent rides
```
Comment (DomLaw82) [inline README.md:39]:
```suggestion
2. Parses logs and cleans data, passes into SQLite table ready for querying from the Live Dashboard, displaying recent and current pages
```

File: README.md [hunk 2]
```diff
@@ -20,13 +22,47 @@ These take the form of:
...
+3. At any point cleaned data suggests current user exceeds heart rate limit during ride, email alert trigger sent using Amazon simple email service (SES)
```
Comment (DomLaw82) [inline README.md:40]:
```suggestion
3. At any point current user heart rate exceeds heart rate limit during the ride, calculated as a function of the user's age, email alert trigger sent using Amazon simple email service (SES)
```

File: README.md [hunk 3]
```diff
@@ -20,13 +22,47 @@ These take the form of:
...
+4. For a given ride for a specific user, at the end of their current ride, two csv files (user_info & ride_info) sent to s3 bucket (file storage service)
+5. AWS Lambda function reads csv files from s3 bucket and extracts key metrics (using Pandas) for long term storage in AWS RDS postgres table
```
Comment (DomLaw82) [inline README.md:42]:
```suggestion
5. AWS Lambda function reads csv files from s3 bucket and extracts key metrics (using Pandas) for long term storage in AWS RDS PostgreSQL table
```

File: README.md [hunk 4]
```diff
@@ -20,13 +22,47 @@ These take the form of:
...
+6. RDS comprised of two tables, users and rides, joined by user_id
+7. Tableau dashboard connected to RDS schemas and visuals created from it, live streaming the information
+8. API hosted on EC2 performing SQL queries from RDS
```
Comment (DomLaw82) [inline README.md:45]:
```suggestion
8. API hosted using AWS API Gateway, integrated with AWS Lambda Functions
```

File: README.md [hunk 5]
```diff
@@ -20,13 +22,47 @@ These take the form of:
...
 
-blah
+### Daily Report generator
 
-## Setup
+### Heart Rate Email Alerts
 
-blah
+1. EC2 hosting Kafka data stream automatically alerts user when their heart rate exceeds a certain level
+2. Heart Rate Level calculated by age and weight
```
Comment (DomLaw82) [inline README.md:68]:
```suggestion
2. Heart Rate Level calculated by age
```

**Review by DomLaw82 — APPROVED** [review]
Just some slight changes suggested with the wording

File: README.md [hunk 6]
```diff
@@ -20,22 +29,93 @@ These take the form of:
...
+## Project Roadmap
 
-blah
+![Screenshot_Excalidraw](https://user-images.githubusercontent.com/115073814/217223449-26e6a315-d01c-4a11-a743-ceee6c4e60d7.png)
 
-## Setup
+### Summary of above roadmap:
 
-blah
+1. Kafka Consumer in EC2 takes logs from Kafka Data stream every second
```
Comment (sebjshaw) [inline README.md:38]:
```suggestion
1. Kafka consumer polls the Deloton topic receiving two logs per second. Combines these logs into one containing all the data for that second of the ride
```

File: README.md (hunk 2 above)
Comment (sebjshaw) [reply README.md:40]:
```suggestion
3. 3. Any time the current user's heart rate exceeds their heart rate limit during the ride, calculated as a function of their age, an email alert trigger is sent using Amazon simple email service (SES)
```

File: README.md [hunk 7]
```diff
@@ -20,22 +29,93 @@ These take the form of:
...
 
+Resources used:
+- [Pandas](https://pandas.pydata.org/)
+- [Amazon Web Services](https://aws.amazon.com/)
+    - API Gateway
+    - EC2
+    - ECR
+    - Lambda
+    - S3
+    - RDS
```
Comment (sebjshaw) [inline README.md:98]:
```suggestion
    - RDS
    - SES
```

File: README.md [hunk 8]
```diff
@@ -20,22 +29,117 @@ These take the form of:
...
+Get all rides for a rider with a specific ID
+
+```DELETE``` ```/ride/:id```
+Delete a with a specific ID
+
+```GET``` ```/daily```
+Get all of the rides in the current day
+
+```GET``` ```/daily?date=01-01-2020```
+Get all rides for a specific date
```
Comment (sebjshaw) [inline README.md:81]: nice addition of all the API endpoints

Comment (AlexSkowronski2) [reply README.md:81]: I definitely didn't just nick that from the curriculum

**Pull request merged**